### DISCUSSION TITLE
L10n directionality

### DISCUSSION THREAD

**Discussion by Faaatman** [flyerhq/flutter_chat_ui#237] [Ideas & Requests]
The ability for the chat direction to change based on the current localization.
Ex: LTR if English and RTL if Arabic.
Right now only the bottom input is changed but the padding is incorrect and the chat bubbles stay the same.

edit: I have created a PR #238 that fixes this issue

**Comment by demchenkoalex** [comment]
Merged and released in `v1.5.6`. Thank you for the PR!
